haris2055/SOEN390
# Password Reset Email & Change Password Features

This pull request involves the full implementation of the following user stories: #84 (Forgot Password Reset) and #85 (Change Password). 

#84 (Forgot Password Reset) feature accessible from the 'click here' link on the login page
To test this feature: 
- go to login page
- click on 'click here' after 'Forgot Password?'
- enter your email (the email should be registered to an account); if you do not already have an account assigned to your email, register one and test the feature after activating your account)
- go to inbox, click on link 
- create new password & submit the form 
- log in 

#85  (Change Password) feature accessible from the settings page when logged in
To test this feature: 
- log in to your user account 
- go to settings 
- fill out the form and submit 

This pull request corrects bug #156. The popup alert is replaced with a success message that appears on the screen. 

This pull request closes #84 , closes #85, closes #156 and closes #71.
